spec/models/preserved_copy_spec.rb

```diff
@@ -194,4 +194,18 @@
       expect(preserved_copy.matches_po_current_version?).to be false
     end
   end
+
+  context '.least_recent_version_audit(last_checked_b4_date, storage_dir)' do
+    let(:db_query) {
+      described_class.least_recent_version_audit(Time.now.utc, 'spec/fixtures/storage_root01/moab_storage_trunk')
+    }
+
+    it 'returns an ActiveRecord::Relation' do
+      expect(db_query).to be_a(ActiveRecord::Relation)
+    end
+
+    it 'returns the preserved_copy in the db' do
+      expect(db_query).to include(preserved_copy)
```

Comment:
What I'm trying to express is ... can/should these tests be more explicit in how you get to preserved copies from ActiveRecord::Relation?

The calling code does:

```ruby
 pcs = PreservedCopy.least_recent_version_audit(last_checked_b4_date, storage_dir)
```

so do we want to be explicit that we can view the ActiveRecord relation as an Array of PreservedCopy objects?

Maybe that would go with the preview test (line 204) or in a separate test ... or nowhere because I'm so ignorant about ActiveRecord relations.   (second cousin?  first cousin once removed?  🤔 )

Reply:
I added one commit that tries to check that dates come in oldest -> newest order, and took it that we already have specs testing that -- null value for last_version_audit gets done before a dated value and that -- dates AFTER the date arg are not included. Since a couple of the tests are similar, I tried to wrap them up in a shared context block, and extract some `let` statements to cut down on the verbosity.

I also switched the preserved_copy spec to use rspec's ActiveRecord::Relation `match_array` matcher (https://relishapp.com/rspec/rspec-rails/docs/matchers/activerecord-relation-match-array), which hopefully addresses the concern about making it clear how you get to preserved copies from an ActiveRecord::Relation.